README.md

```diff
@@ -61,6 +61,8 @@ Node.js based automation frameworks locally and or in Sauce Labs.
         ![WebdriverIO - Webdriver - Examples - W3C](https://github.com/saucelabs-training/demo-js/workflows/WebdriverIO%20-%20Webdriver%20-%20Examples%20-%20W3C/badge.svg)
       - [WebdriverIO with Typescript](webdriverio/webdriver/examples/typescript)\
         ![WebdriverIO - Webdriver - Examples - TS](https://github.com/saucelabs-training/demo-js/workflows/WebdriverIO%20-%20Webdriver%20-%20Examples%20-%20TS/badge.svg)
+      - [WebdriverIO with Deque Axe Accessibility](webdriverio/webdriver/examples/deque)\
+        ![WebdriverIO - Webdriver - Examples - Deque Axe Accessibility](https://github.com/saucelabs-training/demo-js/actions/workflows/wdio.webdriver.examples.deque.axe.yml/badge.svg)]
```

Comment:
I don't see the workflow badge in GH actions in this PR, so it's better to remove it here

```suggestion
```

Reply:
Lol, you already merged the workflow file. Can you please remove that. We are not testing on the existence of the accessibility report so having the workflow file has no added value

Reply:
I checked out the performance package-lock.json into my branch from main to revert whatever weird changes I made. Sorry.

Reply:
NP my friend, looks good now. I've just remove one last service that I forgot to check, but it's ok now.

Thanks